packages/nameguard-react/src/components/LoadingReportIcon/LoadingReportIcon.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { Tooltip } from "../Tooltip/Tooltip";
+import { ENSName } from "@namehash/ens-utils";
+import { RatingIcon, RatingIconSize } from "../Report/RatingIcon";
+import { RatingLoadingIcon } from "../icons/RatingLoadingIcon";
+
+type LoadingReportIconProps = {
+  onClickOverride?: (ensName: ENSName) => void;
+  size?: RatingIconSize;
+} & React.ComponentProps<typeof RatingIcon>;
+
+export const LoadingReportIcon = ({
```

Comment:
Should we rename this to something such as `ReportLoadingIcon` to have more consistency with how we name `RatingLoadingIcon`?